internal/cni-server/cni-plugin.go

```diff
@@ -124,7 +124,7 @@ func (s *server) CmdDelete(args *skel.CmdArgs) (err error) {
 	delete(s.listeners, inode)
 
 	s.Unlock()
-	m, err := ebpf.LoadPinnedMap(path.Join(s.bpfMountPath, "mark_pod_ips_map"), &ebpf.LoadPinOptions{})
+	m, err := ebpf.LoadPinnedMap(path.Join(s.bpfMountPath, "mark_pod_ips"), &ebpf.LoadPinOptions{})
```

Comment:
For the cleanup `local_pod_ips` map I wanted to use `inum`, but not sure how to make it with 1 access. Do we want to first get IP for inum and later remove entry?

Reply:
Unfortunately we are not able to do it as in `mb_connect` we are getting values from `local_pod_ips` map not only for the IP of current pod (current netns) but as well as for destination IP address, so we will have to clean both of the maps, where in `local_pod_ips` the key is the IP address and in the `mark_pod_ips` which I just renamed to `netns_pod_ips` where the key is the netns inum